tooling/miniutil/src/fmt/vtables.rs

```diff
@@ -0,0 +1,32 @@
+use super::*;
+
+pub(super) fn fmt_vtables(vtables: Map<VTableName, VTable>) -> String {
+    let mut out = String::new();
+
+    let mut vtables: Vec<(VTableName, VTable)> = vtables.iter().collect();
+
+    // The vtables are formatted in the order of their names.
+    vtables.sort_by_key(|(VTableName(name), _)| *name);
+
+    for (vname, vtable) in vtables {
+        out += &fmt_vtable(vname, vtable);
+    }
+    out
+}
+
+fn fmt_vtable(vname: VTableName, vtable: VTable) -> String {
+    let mut out = format!("vt{id} {{\n", id = vname.0.get_internal());
```

Comment:
```suggestion
    let mut out = format!("vtable{id} {{\n", id = vname.0.get_internal());
```
No need to use unreadable mnemonics. ;)